ddtrace/tracer.py

```diff
@@ -102,7 +132,8 @@ def _default_span_processors_factory(
     single_span_sampling_rules,  # type: List[SpanSamplingRule]
     agent_url,  # type: str
 ):
-    # type: (...) -> List[SpanProcessor]
+    # type: (...) -> Tuple[List[SpanProcessor], Optional[Any]]
+    # FIXME: type should be AppsecSpanProcessor but we have a cyclic import here
```

Comment:
the `#FIXME` is not needed anymore, right?

Reply:
The FIXME is about the typing with Any above.